### PR TITLE
Feature cover image to songs

### PR DESCRIPTION
**Pull request by condef5**

### CODE REVIEW

File: app/assets/stylesheets/home.scss [hunk 1]
```diff
@@ -14,11 +14,28 @@ a[data-method='delete']:hover {
...
-  font-size: 14px;
+  background-color: #24292e;
   a {
-    border-bottom: 1px solid #453886;
+    color: #fff;
     padding-bottom: 0.25em;
     text-decoration: none;
+    transition: all 0.5s;
+    position: relative;
+    &:before {
```
Comment (Linzeur) [inline app/assets/stylesheets/home.scss]: Can you explain me what "&:before" does?

File: app/views/admin/songs/_form.html.erb [hunk 2]
```diff
@@ -21,9 +21,14 @@
 
   <div class="field">
     <%= f.label :rating %>
-    <%= f.text_field :rating %>
+    <%= f.number_field :rating %>
```
Comment (Linzeur) [inline app/views/admin/songs/_form.html.erb]: As reminder, the rating field shouldn't be shown both create and update

File: config/environments/development.rb [hunk 3]
```diff
@@ -29,7 +31,6 @@
     config.cache_store = :null_store
   end
 
-  # Store uploaded files on the local file system (see config/storage.yml for options)
   config.active_storage.service = :local
```
Comment (Linzeur) [inline config/environments/development.rb]: Be careful, you repeat this line in the line 9.

File: app/assets/stylesheets/home.scss (hunk 1 above)
Comment (condef5) [reply app/assets/stylesheets/home.scss]: The `&` operator refers to the current selector.

File: app/views/admin/songs/_form.html.erb (hunk 2 above)
Comment (luciagirasoles) [reply app/views/admin/songs/_form.html.erb]: if rating is not updated by the user, it should show a message like "none has voted" (or sth like that)

**Comment by condef5** [comment]
Close #43

**Pull request merged**